### DISCUSSION TITLE
No need to convert BGR to RGB?

### DISCUSSION THREAD

**Discussion by GOBish** [STVIR/pysot#390] [General] [answered]
Just wondering, the demo.py takes in the image to openCV and does all the tracking as is.

Is there no need to convert from BGR colorspace, which OpenCV uses, to RGB - which perhaps the model was trained on?
(using color_image = cv2.cvtColor(color_image, cv2.COLOR_BGR2RGB) )

Thanks

**Comment by ZhiyuanChen** [comment, marked as answer]
No.
